packages/block-editor/src/components/block-inspector/style.scss

```diff
@@ -1,3 +1,17 @@
+.block-editor-block-inspector {
```

Comment:
I can see how this simplifies things but does it still hold true if we decide to show the inspector in a Modal tomorrow instead of a sidebar panel? Will we need these styles as well in the modal?

Reply:
Makes sense, both these styles also seem to be something that can be absorbed by the UI components themselves.